index.html

```diff
@@ -1,86 +1,1369 @@
-      rel="stylesheet"
-      crossorigin="anonymous"
-    />
-    <!-- Bootstrap Icons -->
-    <link
-      rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.10.5/font/bootstrap-icons.css"
-    />
-    <!-- Docs -->
-    <link href="./assets/docs.css" rel="stylesheet" />
-  </head>
-  <body>
-    <div class="header">
-      <div class="container-md">
-        <h1>Free Country Flags in SVG</h1>
-        <p class="lead">
-          A curated collection of all country flags in SVG — plus the CSS for
-          easier integration.
-        </p>
-        <p class="lead">
-          All flags are in <strong>4x3</strong> and <strong>1x1</strong> formats
-          only.
-        </p>
-        <pre class="install">yarn add flag-icons</pre>
-        <p>For more instructions check out on GitHub.</p>
-        <a class="btn btn-light me-2" href="https://github.com/lipis/flag-icons"
-          ><i class="bi bi-github"></i> View on GitHub</a
-        >
-        <a
-          class="btn btn-light"
-          href="https://github.com/lipis/flag-icons/archive/main.zip"
-          ><i class="bi bi-cloud-arrow-down-fill"></i> Download Flags</a
-        >
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>flag-icons: Free Country Flags in SVG</title>
+
+  <!-- Google tag (gtag.js) -->
+  <script
+    async
+    src="https://www.googletagmanager.com/gtag/js?id=G-9LZGY0J38Z"
+  ></script>
+  <script>
+    window.dataLayer = window.dataLayer || [];
+    function gtag() {
+      dataLayer.push(arguments);
+    }
+    gtag("js", new Date());
+    gtag("config", "G-9LZGY0J38Z");
+  </script>
+
+  <!-- Meta tags -->
+  <meta property="og:title" content="flag-icons: Free Country Flags in SVG">
+  <meta property="og:type" content="website">
+  <meta property="og:url" content="https://flagicons.lipis.dev">
+  <meta property="og:description" content="A curated collection of all country flags in SVG">
+  <meta property="og:image" content="https://flagicons.lipis.dev/assets/og.png">
+  <link rel="canonical" href="https://flagicons.lipis.dev">
+  <link rel="icon" href="/favicon.ico">
+
+  <!-- Main styles -->
+  <style>
+    :root {
+      --primary: #2b8a3e;
+      --primary-dark: #237032;
+      --primary-light: #8ce99a;
+      --primary-ultra-light: #ebfbee;
+      --secondary: #e8590c;
+      --secondary-light: #ffad8a;
+      --dark: #212529;
+      --light: #f8f9fa;
+      --gray: #6c757d;
+      --light-gray: #e9ecef;
+      --border-radius: 8px;
+      --shadow: 0 4px 6px rgba(0, 0, 0, 0.1);
+      --transition: all 0.2s ease;
+      
+      /* Code syntax colors */
+      --code-bg: #1e1e1e;
+      --code-text: #d4d4d4;
+      --code-keyword: #569cd6;
+      --code-string: #ce9178;
+      --code-tag: #569cd6;
+      --code-attribute: #9cdcfe;
+      --code-value: #ce9178;
+      --code-comment: #6a9955;
```

Comment:
I'm in love with the open-colors.. could you use any of those? https://yeun.github.io/open-color/